cypress-tests/cypress/e2e/PaymentMethodListTest/00000-PaymentMethodListTests.cy.js

```diff
@@ -1,6 +1,7 @@
 import apiKeyCreateBody from "../../fixtures/create-api-key-body.json";
 import createConnectorBody from "../../fixtures/create-connector-body.json";
 import merchantCreateBody from "../../fixtures/merchant-create-body.json";
+import customerCreateBody from "../../fixtures/create-customer-body.json";
```

Comment:
Why are we doing this when we clearly have `imports.js` which we're SUPPOSED to be using?

Reply:
Everywhere fixtures.body is not being used , do you want me change everywhere?

Reply:
in payments and payouts, we're indeed using the import file:

```js
import * as fixtures from "../../fixtures/imports";`
...
... fixtures.<body_name> ...
```

so, yes, we should change it.